frontend/src/common/components/mentoringStepper/MentoringStepper/MentoringStepper.tsx

```diff
@@ -0,0 +1,183 @@
+import styled from '@emotion/styled';
+
+import tooltipIcon from '../../../assets/images/tooltip.svg';
+import {
+  MentoringReservationStatusTypeEnum,
+  type MentoringReservationStatusType,
+} from '../../../types/mentoringReservationStatus';
+import Step from '../Step/Step';
+
+interface MentoringStepperProps {
+  status: MentoringReservationStatusType;
+}
+
+const statusTextMap = Object.values(MentoringReservationStatusTypeEnum).reduce(
+  (acc, key) => {
+    switch (key) {
+      case 'PENDING':
+        acc[key] = '예약신청';
+        break;
+      case 'APPROVED':
+        acc[key] = '신청확정';
+        break;
+      case 'COMPLETE':
+        acc[key] = '멘토링완료';
+        break;
+    }
+    return acc;
+  },
+  {} as Record<string, string>,
+);
```

Comment:
p4: reduce 좋다 👍👍 

혹시 배열로 만드는 게 아니라 다시 객체로 만드는 거라 이렇게 만드는 건 어때?

 ```suggestion
const statusTextMap = {
  [MentoringReservationStatusTypeEnum.PENDING]: '예약신청',
  [MentoringReservationStatusTypeEnum.APPROVED]: '신청확정',
  [MentoringReservationStatusTypeEnum.COMPLETE]: '멘토링완료',
} as const;
```

Reply:
저렇게 했는데 객체로 만드는게 더 간단하다 생각해서 밑에는 그렇게 했다가 수정을 안했네😅 훨씬 좋은거 같아 수정할게!

Reply:
수정커밋: [refactor: statusTextMap을 reduce에서 객체 리터럴로 변경](https://github.com/woowacourse-teams/2025-Fit-toring/pull/611/commits/edf01706cf21e6b2ada67383aa7fd6c1f3d1456a)